easybuild/easyconfigs/s/Six/Six-1.11.0-GCCcore-6.4.0-Python-2.7.14-bare.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'Six'
```

Comment:
Should be `six`, cfr. http://six.readthedocs.io/: `The name, “six”, comes from the fact that 2*3 equals 6.`